src/App.css

```diff
@@ -6,3 +6,46 @@
 ul {
   list-style: none;
 }
+
+button {
+  width: 12%;
+  max-width: 80px;
+  min-width: 100px;
+  height: 40px;
```

Comment:
your max-width is less than your min-width here; your min will cancel out your max. You can probably ditch the max-width